Added pre-commit hook

ternaus opened the pull request.

albu commented on README.md:
````diff
@@ -218,23 +218,25 @@ Python and library versions: Python 3.7.3 | Anaconda, numpy 1.16.4, pillow 6.0.0
+```
+pytest
+```
+7. Push code to your forked repo.
````
flake8?

ternaus replied:
Automatically evaluated by a pre-commit hook.

albu commented on README.md:
````diff
@@ -218,23 +218,25 @@ Python and library versions: Python 3.7.3 | Anaconda, numpy 1.16.4, pillow 6.0.0
+pip install pre-commit
+pre-commit install
+```
+4. Modify it.
````
modify pre-commit hook? how?

ternaus replied:
I meant, make desired modification in the code. Not in the pre-commit hook.

Rephrased.

ternaus commented on README.md:
````diff
@@ -218,23 +218,25 @@ Python and library versions: Python 3.7.3 | Anaconda, numpy 1.16.4, pillow 6.0.0
+3. Install pre-commit hook:
+```
+pip install pre-commit
+pre-commit install
````
Fixed.

ternaus commented:
1. Added Pre-commit hook for black.
2. Moved text with contributions to contributions.rst
3. Fixed First/Last names in docs.
4. Pre-commit hook was enabled while [3] => conf.py was updated to black standards.

Dipet commented on README.md:
````diff
@@ -218,23 +218,8 @@ Python and library versions: Python 3.7.3 | Anaconda, numpy 1.16.4, pillow 6.0.0
-   flake8
-   ```
+
+To create pull request to the repsitory follow the the documentation at [docs/contributing.rst](docs/contributing.rst)
````
Please fix "follow the the"

ternaus replied:
Thanks!
